index.html

```diff
@@ -16,7 +16,7 @@
     <link rel="icon" type="image/png" sizes="32x32" href="images/favicon.ico/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="96x96" href="images/favicon.ico/favicon-96x96.png">
     <link rel="icon" type="image/png" sizes="16x16" href="images/favicon.ico/favicon-16x16.png">
-    <link rel="manifest" href="images/favicon.ico/manifest.json">
+    <link rel="manifest" href="/manifest.json">
```

Comment:
It's good that you have separated the manifest.json as it isn't explicitly tied to the favicons
How about changing the directory name from **favicon.ico** to **favicons** and effect changes across all references.